dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/domain/ServiceDO.java

```diff
@@ -0,0 +1,43 @@
+package org.apache.dubbo.admin.domain;
+
+/**
+ * @author zmx ON 2018/7/20
```

Comment:
`@author` is not suggested to include in source code

Reply:
Also need LICENSE info

Reply:
thanks，I'll remove author info and add license